lib/extension/window.js

```diff
@@ -1614,10 +1611,10 @@ export class WindowManager extends GObject.Object {
     let nodeWindow;
     nodeWindow = this.tree.findNodeByActor(actor);
 
-    if (nodeWindow) {
+    if (nodeWindow && nodeWindow.isWindow()) {
       this.tree.removeNode(nodeWindow);
       this.renderTree("window-destroy-quick", true);
-      this.removeFloatOverride(nodeWindow, false, true);
+      this.removeFloatOverride(nodeWindow.nodeValue, true);
```

Comment:
Has the signature of this function changed it just the variable name?

Reply:
The signature of removeFloatOverride has been changed in another patch and hasn't been changed here

Reply:
Hey @bennypowers, do you have any concerns on the function signature change?

Hey @p1gp1g, I can update your role so you can merge. Just need someone to test looks OK or need evidence (screencap) that this is working fine.

Reply:
No the signature change doesn't bother me; it's just not immediately clear from this PR why this change was made *here*

If OP wants bonus marks, split that line into a separate commit with a detailed message

But that's not something which should hold up the merge.

If other reviewers have run this and found it acceptable, I'm good with it

Reply:
Alright, just found out that I can add per repository role maintain access to all of you in settings.